molecule/not_debian_11/molecule.yml

```diff
@@ -75,6 +81,36 @@ provisioner:
       # underscores when creating the group name.
       force_valid_group_names: always
   inventory:
+    group_vars:
+      force_python3:
+        # auto_legacy is still the default behavior until Ansible 2.12
+        # is released, but Molecule overrides this and forces the use
+        # of auto.
+        #
+        # In auto mode Ansible will prefer the Python interpreter
+        # specified in an internal map.  If there is no entry in the
+        # internal map then Ansible will search an ordered list of
+        # potential Python locations and use the first Python
+        # executable from that list that it finds.  At the top of that
+        # list is /usr/bin/python.
+        #
+        # In most cases the internal map specifies /usr/bin/python3,
+        # but neither Debian 11 nor Kali Linux are listed in the
+        # internal map.  Since the installation of amazon-efs-utils
```

Comment:
Is this still needed for the `not_debian_11` scenario? That appeared to pass in https://github.com/cisagov/ansible-role-amazon-efs-utils/runs/1495039044?check_suite_focus=true when the `debian11` scenario failed.

Reply:
I'm just keeping the `molecule.yml` files for the more specific scenarios in sync with the ones for the `default` scenario.  The extra definitions do no harm since they aren't applied.